From d7dcacd764a96d9d9ffe1528f39bb90e6c55afd0 Mon Sep 17 00:00:00 2001
From: Kylie McClain <kylie@somas.is>
Date: Fri, 5 Sep 2025 01:56:30 -0400
Subject: [PATCH] Allow for setting a custom `pass` path

---
 pass.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pass.cpp b/pass.cpp
index 12b9856..5fcc792 100644
--- a/pass.cpp
+++ b/pass.cpp
@@ -234,7 +234,7 @@ void Pass::run(const KRunner::RunnerContext &context, const KRunner::QueryMatch
         args << "otp";
     }
     args << "show" << match.text();
-    pass->start("pass", args);
+    pass->start(NIXPKGS_PASS, args);
 
     connect(pass, static_cast<void(QProcess::*)(int, QProcess::ExitStatus)>(&QProcess::finished),
             [=](int exitCode, QProcess::ExitStatus exitStatus) {
-- 
2.50.1

